recipe/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sarif_om" %}
+{% set version = "1.0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: cd5f416b3083e00d402a92e449a7ff67af46f11241073eea0461802a3b5aef98
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
+
+requirements:
+  host:
+    - pip
+    - python
+    - wheel
+    - setuptools
+    - pbr ==5.6.0
```

Comment:
pbr and attrs shouldn't be pinned. Also, I don't think they should be in the host section. I guess these got put in by grayskull or something?
See the deps [here](https://github.com/microsoft/sarif-python-om/blob/fa8028a06087d039a1f4d96034b7961531905dc1/setup.cfg#L21). It's in the install_requires section, which means, packages that need to be installed at runtime.
Also worth having a read of this if you haven't already: https://github.com/anaconda-distribution/perseverance-skills/blob/main/sections/02_Package_building/02_Recipes/Pinnings.md
And let me know if you have any questions :)

Reply:
Just to update, pbr not being in run was tested but didn't make it all the way through tests, I've kept it in as is for the time being but of course will revisit if that is incorrect 👍

Reply:
Because it's in install_requires, it fails pip check if not included.
setup.py could be patched but it's not worth the effort in my opinion.